cfpb/sbl-frontend
# [E2E] Get tests to consistently pass in `headless` mode

Closes #987 

The failures I've experienced seem to be isolated to me + my machine, and most consistently in `headless` mode, but this workaround is implemented in a way that should not interfere with others' consistently passing tests. 

## Changes

- Implements retries when clicking a link that navigates to a new domain for the first time

## How to test this PR

1. Start local SBL Platform
2. `yarn playwright test --workers 4`
3. Ensure all tests pass

## Screenshots

![screencapture-localhost-9323-2024-10-30-13_31_18](https://github.com/user-attachments/assets/4ace1a05-fd75-4632-8502-4f2f9f1f6806)
